### DISCUSSION TITLE
Is it possible to run an openvpn connection in proxy mode?

### DISCUSSION THREAD

**Discussion by Pinkolik** [schwabe/ics-openvpn#1506] [Q&A]
Hello! I want to run an openvpn connection in proxy mode in order to make it a part of a proxy chain but I couldn't figure out how. Is there such feature? And if not, are you planning on it?

**Comment by schwabe** [comment]
The app is a VPN, not really a proxy.

**Reply by Pinkolik** [reply]
Yeah, I understand that but limitations of Android don't allow to use two "VPN"s at the same time. I'm trying to achieve device -> VPN -> Orbot -> Internet configuration and there's really no other way I can think of (without rooting device).  There is a hack which exploits work profile functionality https://itsignacioportal.github.io/netguard-pdnsf-any-vpn-combo/ but I hope there is a much easier and convenient way to do it. Other apps that use Android's VPN functionality have proxy mode (Orbot, SagerNet), so I guess it could be possible to have that with OpenVPN.
P.S. There's also a plugin for WireGuard in SagerNet but I'd prefer OpenVPN
